CMakeLists.txt

```diff
@@ -71,19 +76,11 @@ if(WITH_COVERAGE)
     setup_target_for_coverage_lcov(NAME coverage EXECUTABLE ctest)
 endif()
 
-# ######################################################################################################################
-# Includes
-# ######################################################################################################################
-
-include_directories(include)
-
 # ######################################################################################################################
 # Targets
 # ######################################################################################################################
 
-add_subdirectory(src/common)
```

Comment:
* keep it as is;
* do not add cmake file to src: this lib consists of 4 main libs: common, iam, sm, and will have um, cm in the feature. Basically, client of this lib requires only one part of the lib (+common). For example: iam_cpp will include common and iam only, sm_cpp will include common and sm etc.

Reply:
Done